app/Http/Controllers/Backend/AccountOptionController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Http\Controllers\Backend;
+
+use App\Http\Controllers\Controller;
+use Exception;
+use Illuminate\Support\Str;
+use App\Http\Controllers\BaseController;
+use App\Http\Requests\AccountOptionRequest;
+use App\Services\AccountOptionService;
+use Illuminate\Http\Request;
+use Illuminate\Http\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+
+class AccountOptionController extends BaseController
+{
+    public function __construct(private readonly AccountOptionService $accountOptionService)
+    {
+    }
+
+    /**
+     * @OA\GET(
+     *     path="/api/v1/backend/AccountOptions",
```

Comment:
path should be always url friendly, instead of this -
`AccountOptions` to `account-options`

Reply:
Please change same in other places and definitely after change make a quick test must.